com.unity.netcode.gameobjects/TestHelpers/Runtime/NetcodeIntegrationTest.cs

```diff
@@ -431,6 +431,18 @@ protected virtual void OnNewClientStartedAndConnected(NetworkManager networkMana
         {
         }
 
+        /// <summary>
+        /// CreateAndStartNewClient Only
+        /// Override this method to bypass the waiting for a client to connect.
+        /// </summary>
+        /// <remarks>
+        /// Use this for testing connection and disconnection scenarios
+        /// </remarks>
+        protected virtual bool WaitForNewClientToConnect(NetworkManager networkManager)
```

Comment:
This feels like it should be a property so it's more obvious the intent. When I first looked at this, I thought "this is just returning true, it's not waiting for a new client to connect at all".